pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -2454,7 +2454,11 @@ def _get_start_value(self, node: nodes.NodeNG) -> tuple[int | None, Confidence]:
             and isinstance(node.operand, (nodes.Attribute, nodes.Name))
         ):
             inferred = utils.safe_infer(node)
-            start_val = inferred.value if inferred else None
+            start_val = None
+            if inferred and isinstance(inferred, nodes.Const):
```

Comment:
```suggestion
            if isinstance(inferred, nodes.Const):
```

The `is not None` check is included in the `isinstance`. We can also keep this a one-liner but just use `isinstance`.

Reply:
Ha, clever !